vagrant/Vagrantfile

```diff
@@ -146,29 +175,38 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       nodes: ["master1", "node1", "node2"],
     }
 
+    if deployment_type == 'origin'
+      ansible_groups[:'OSEv3:vars'][:openshift_release] ='latest'
+    end
+    print("deployment_type = %s" % (deployment_type))
+
     ansible_host_vars = {
       master1:  {
+        ansible_ssh_common_args: '"-o StrictHostKeyChecking=no"',
```

Comment:
Ah, so it turns out I broke this by moving the sync path to `..` instead of `.`.  Not entirely sure now why I did that.  Looking :eyes: